controllers/topology/internal/extensions/patches/inline/json_patch_generator.go

```diff
@@ -162,25 +162,29 @@ func patchIsEnabled(enabledIf *string, variables map[string]apiextensionsv1.JSON
 
 // jsonPatchRFC6902 is used to render the generated JSONPatches.
```

Comment:
seems this changes were included by accident .

Reply:
@enxebre Yes. I struggled a bit with this. It looks like the squash is finally done.